src/intervals/intervals.jl

```diff
@@ -5,37 +5,34 @@
 
 ## Interval type
 
+const validity_check = false
```

Comment:
Another reason to change this default is the fact that many users may construct directly the intervals using the constructor instead of `interval`...